modules/ROOT/pages/design-change-root-file.adoc

```diff
@@ -0,0 +1,80 @@
+
+For example, suppose that a RAML API-specification project contains two specifications, each including (with the `!include` keyword) a different set of API fragments that are also in the project. When you publish the project to Exchange, all of the files in the project are published. However, Exchange needs to validate the specification and the files that it includes. Without a root file, Exchange cannot ascertain which file to parse first and which set of files to parse.
+
+To take another example, suppose that you want to publish an API-fragment project that contains several fragments, one of which is independent and the remaining files of which are a library fragment and its dependencies. If no root file is specified in the project, Exchange cannot parse the files correctly. In this case, you could either set the independent fragment as the root file, indicating to Exchange that the file can be parsed alone. Or you could set the library file as the root file, indicating to Exchange that parsing should begin with that file and proceed to the dependent fragments.
+
+[NOTE]
+====
+The first time that you open in the text editor a project that was created before the February 8, 2020 release of API Designer, you are prompted to set a root file.
+====
+
+== Root Files in API-Specification Projects in the Text Editor
+
+Every API-specification project is created with a default root file. You can switch the root file when there is more than one specification in a project.
+
+=== The Default Root File
+
+When you create an API-specification project, the file that is created by default is set to be the root file:
+
+image:apid-default-root-file-api-spec-project.png[The default root file in an API-specification project named "Orders API".]
+
+<1> The name of the RAML API-specification project is `Orders API`.
+<2> The default file is named `orders-api.raml` and is set as the root file because every API-specification project must have a root file.
+
+== Setting the Root File in an API-Specification Project That Contains More Than One Specification
+
+When you import a specification into a project, API Designer asks whether you want to set the specification as the root file of the project.
+
+You can set any specification in an API-specification project as the root file of that project. For example, the following image shows that a project contains two specifications:
+
+image:apid-root-file-api-spec-project-multi-specs.png[An API-specification project with two specifications. The current root file is the RAML version of the specification.]
+
+<1> The OAS version of the Inventory Process API specification.
```

Comment:
Notice that the <1> <2> items are a mixture of fragments and sentences, and all end with periods. Any way to make them parallel?